torchvision/datasets/flowers102.py

```diff
@@ -0,0 +1,116 @@
+from pathlib import Path
+from typing import Any, Tuple, Callable, Optional
+
+import numpy as np
+import PIL.Image
+
+from .utils import verify_str_arg, download_and_extract_archive, download_url
+from .vision import VisionDataset
+
+
+class Flowers102(VisionDataset):
+    """`Oxford 102 Flower <https://www.robots.ox.ac.uk/~vgg/data/flowers/102/>`_ Dataset.
+
+    .. warning::
+
+        This class needs `scipy <https://docs.scipy.org/doc/>`_ to load target files from `.mat` format.
+
+    Oxford 102 Flower is an image classification dataset consisting of 102 flower categories. The
+    flowers chosen to be flower commonly occurring in the United Kingdom. Each class consists of
+    between 40 and 258 images.
+
+    The images have large scale, pose and light variations. In addition, there are categories that
+    have large variations within the category and several very similar categories.
+
+    Args:
+        root (string): Root directory of the dataset.
+        split (string, optional): The dataset split, supports ``"train"`` (default), ``"val"``, or ``"test"``.
+        transform (callable, optional): A function/transform that takes in an PIL image and returns a
+            transformed version. E.g, ``transforms.RandomCrop``.
+        target_transform (callable, optional): A function/transform that takes in the target and transforms it.
+    """
+
+    def __init__(
+        self,
+        root: str,
+        split: str = "train",
+        download: bool = True,
```

Comment:
The download flag is missing from the docstring.

Reply:
Added now.